src/views/SignDidStart/SignDidStart.tsx

```diff
@@ -0,0 +1,86 @@
+import { browser } from 'webextension-polyfill-ts';
+
+import { Link, useLocation } from 'react-router-dom';
+
+import { filter } from 'lodash-es';
+
+import { useEffect } from 'react';
+
+import * as styles from './SignDidStart.module.css';
+
+import { Identity } from '../../utilities/identities/types';
+import { IdentitiesCarousel } from '../../components/IdentitiesCarousel/IdentitiesCarousel';
+import { generatePath, paths } from '../paths';
+import { isFullDid } from '../../utilities/did/did';
+import { useIdentityCredentials } from '../../utilities/credentials/credentials';
+import { usePopupData } from '../../utilities/popups/usePopupData';
+import { SignDidOriginInput } from '../../channels/SignDidChannels/types';
+
+interface Props {
+  identity: Identity;
+  onPopupData: (popupData: SignDidOriginInput) => void;
+}
+
+export function SignDidStart({ identity, onPopupData }: Props) {
+  const t = browser.i18n.getMessage;
+
+  const { search: popupQuery } = useLocation();
+
+  const popupData = usePopupData<SignDidOriginInput>();
+  useEffect(() => {
+    onPopupData(popupData);
+  }, [popupData, onPopupData]);
+
+  const { address, did } = identity;
+
+  const credentials = useIdentityCredentials(did);
+
+  const attestedCredentials = filter(
+    credentials,
+    ({ status }) => status === 'attested' || status === 'pending',
+  );
```

Comment:
I just realized that actually this can be even more explicit. Also since we’re not using any of lodash’ helpers, we could do without it.
```suggestion
  const noAttestedCredentials = !credentials.some(
    ({ status }) => status === 'attested' || status === 'pending',
  );
```